packages/trace-viewer/src/traceModernizer.ts

```diff
@@ -395,6 +395,10 @@ export class TraceModernizer {
         result.push({ ...event, monotonicTime: 0, origin: 'library' });
         continue;
       }
+      if (event.type === 'frame-snapshot') {
+        result.push({ ...event, snapshot: { ...event.snapshot, wallTime: 0 } });
```

Comment:
How about we make `wallTime` optional in the type definition, since there is no way we can have it for old traces anyway?